lib/base/model.js

```diff
@@ -584,12 +592,18 @@ ModelBase.prototype.saveMethod = function(options) {
 
   if (options.patch) {
     if (options.method === 'insert')
-      throw new TypeError(`Cannot accept incompatible options: method=insert, patch=${options.patch}`);
+      throw new TypeError(
+        `Cannot accept incompatible options: method=insert, patch=${
+          options.patch
```

Comment:
This is definitely super weird :confused: Why is the `options.patch` in a single line by itself? Is there like a 80 column max line length limit or something?

Reply:
Yes, it wraps per default at 80 columns however you can set it via `printWidth` to a more reasonable amount. I suggest 120 columns. This should fix a few of your aforementioned problems. How many columns do you suggest @ricardograca

Reply:
:+1: 120 sounds good to me as well. 80 is definitely too narrow.

Reply:
@chentsulin Can you please set `printWidth: 120` and run prettier on the codebase again?